tests/test_transformBoundedField.py

```diff
@@ -0,0 +1,171 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""
+Tests for math.TransformBoundedField
+
+Run with:
+   ./testTransformBoundedField.py
+or
+   python
+   >>> import testSchema; testSchema.run()
+"""
+from __future__ import absolute_import, division, print_function
+
+import os
+import unittest
+
+import astshim
+import numpy as np
+from numpy.testing import assert_allclose
+
+
+import lsst.utils.tests
+import lsst.pex.exceptions
+import lsst.afw.geom
+import lsst.afw.image
+from lsst.afw.math import TransformBoundedField
+
+try:
+    type(display)
+except NameError:
+    display = False
+
+CHEBYSHEV_T = [
+    lambda x: x**0,
+    lambda x: x,
+    lambda x: 2*x**2 - 1,
+    lambda x: (4*x**2 - 3)*x,
+    lambda x: (8*x**2 - 8)*x**2 + 1,
+    lambda x: ((16*x**2 - 20)*x**2 + 5)*x,
+]
+
+
+class TransformBoundedFieldTestCase(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        self.longMessage = True
+
+        # an arbitrary bounding box (not that this kind of field cares)
+        self.bbox = lsst.afw.geom.Box2I(lsst.afw.geom.Point2I(-3, 4),
+                                        lsst.afw.geom.Extent2I(5, 30))
+
+        # a list of points contained in the bbox
+        self.pointList = lsst.afw.geom.Box2D(self.bbox).getCorners()
```

Comment:
I believe so. In this case it doesn't matter -- the functional form of the transform has no intrinsic limits; the bbox is arbitrary.